### DISCUSSION TITLE
react-day-picker allows to click days out of range

### DISCUSSION THREAD

**Discussion by July52rus** [gpbl/react-day-picker#1407] [Support]
When you set up Datepicker so

<DayPicker 
  disabledDays = {{
    after: new Date(),
  }}
  showOutsideDays />;

With specifying disabledDays it allows to click on days out of _today_ range.

**Comment by Elytum** [comment]
react-day-picker reports every user interaction, in order to ignore clicks on disabled days, check for the modifier "disabled".

Here's how you can implement it with onDayClick:

  ```
(day: Date, { disabled, selected }: DayModifiers) => {
    if (disabled) return;
    this.setState({
      selectedDay: selected ? undefined : day,
    });
  }
```

**Comment by landsman** [comment]
I found a same problem.

Related: https://github.com/gpbl/react-day-picker/discussions/1584#discussioncomment-3864945
